samples/contrib/aws-samples/mnist-kmeans-sagemaker/mnist-classification-pipeline.py

```diff
@@ -127,39 +129,29 @@ def mnist_classification(
     hpo_strategy="Bayesian",
     hpo_metric_name="test:msd",
     hpo_metric_type="Minimize",
-    hpo_early_stopping_type="Off",
     hpo_static_parameters={"k": "10", "feature_dim": "784"},
     hpo_integer_parameters=[
         {"Name": "mini_batch_size", "MinValue": "500", "MaxValue": "600"},
         {"Name": "extra_center_factor", "MinValue": "10", "MaxValue": "20"},
     ],
-    hpo_continuous_parameters=[],
     hpo_categorical_parameters=[
         {"Name": "init_method", "Values": ["random", "kmeans++"]}
     ],
     hpo_channels=hpoChannels,
-    hpo_spot_instance=False,
-    hpo_max_wait_time=3600,
-    hpo_checkpoint_config={},
-    hpo_max_num_jobs=9,
+    hpo_max_num_jobs=3,
     hpo_max_parallel_jobs=3,
```

Comment:
I think we can put a default value of 3 here

Reply:
removing hpo_max_num_jobs but leaving in parallel_jobs because sometimes customers might have instance limits